scripts/ansible/roles/avalanche_staker/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: Create staking key
+  become: true
+  openssl_privatekey:
+    path: "{{ avalanche_daemon_staking_tls_key }}"
+    size: 4096
+    backup: true
+    owner: "{{ avalanche_daemon_user }}"
+    group: "{{ avalanche_daemon_group }}"
+    mode: u=rw,go=
+
+# This CSR isn't used at all by Gecko. It's a required step when creating a
```

Comment:
If this is not needed afterwards, does it make sense to delete it after we have the certificates created? So we don't have unused zombies laying around.

Also please use `avalanchego` insteado of `gecko` from now on. I know this is for "historical" reasons and we have loads of this ourselves but just as a friendly reminder.

Reply:
It's needed by the role, to keep it idempotent across multiple runs - so running this role twice won't generate the certificate twice.

I'll add a commit that removes any remaining instances of gecko.